.travis.yml

```diff
@@ -17,11 +17,11 @@ before_install:
 rvm:
   - 1.9.3
   - 1.9.2
-  - 1.8.7
-  - ree
   - 2.0.0
   - 2.1.0
   - 2.2.0
+  - 2.3.0
+  - 2.4.0
```

Comment:
Should this be the most recent ruby versions (2.3.6 and 2.4.3)?

Reply:
sure, why not!